certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -761,14 +806,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact Name",
+                                                        "description": "Name of the administrative contact",
                                                         "title": "Admin Contact Name"
                                                     },
                                                     "count": {
                                                         "type": "integer",
                                                         "format": "int32",
-                                                        "description": "Count",
-                                                        "title": "Count"
+                                                        "description": "Count of connected domains sharing admin contact name",

```

Comment:
@RamboV please make sure everything is grammatically correct. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).